js_basics/task3.js

```diff
@@ -0,0 +1,9 @@
+// Конкатенація рядків та шаблонний рядок
+const userA = "Alis";
+const userB = "Max";
+const users = `${userA} and ${userB}`
+
+const greeting = `Hello dear ${users}, we are happy to see you here! `
```

Comment:
Create the same string below but without using template string. Use concatenation opator +. according to the task both approaches should be shown